compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/util/RepoUtils.java

```diff
@@ -151,6 +155,22 @@ public static String getRemoteRepoURL() {
         return PRODUCTION_URL;
     }
 
+    /**
+     * Get the central package URL.
+     *
+     * @param org     package org
+     * @param pkgName package name
+     * @return central package URL
+     */
+    public static String getCentralPackageURL(String org, String pkgName) {
```

Comment:
Related to https://github.com/ballerina-platform/ballerina-lang/pull/30182/#r621129979